cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server_test.go

```diff
@@ -597,9 +543,9 @@ func TestGetAvailablePackageSummaries(t *testing.T) {
 					Namespace: "",
 				},
 			},
-			charts:           []*models.Chart{{Name: "foo"}},
-			expectedPackages: []*corev1.AvailablePackageSummary{},
-			statusCode:       codes.Unimplemented,
+			expectDBQuery: false,
```

Comment:
This is the test that was previously setting a mock expectation that was never met until the subsequent test tried to run (it just happened to match as an internal error is returned if the expectation doesn't match).